website/docs/docs/collaborate/manage-access/set-up-sso-saml-2.0.md

```diff
@@ -231,11 +245,12 @@ Use this section if you are configuring Google as your identity provider.
 4. Enter the following values on the **Service Provider Details** window:
    - **ACS URL**: `https://cloud.getdbt.com/complete/saml`
    - **Audience URI (SP Entity ID)**: `https://cloud.getdbt.com/`
-   - **Start URL**: (if needed)
-5. The default **Name ID** is the primary email. Multi-value input is not supported.
-6. Use the **Attribute mapping** page to map your organization's Google Directory Attributes to the format that
+   - **Start URL**: `<login slug`
+5. Check the **Signed response** checkbox.
```

Comment:
changing word to adhere to [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/mwong-styleguide-add/contributing/content-style-guide.md)

```suggestion
5. Select the **Signed response** checkbox.
```